scmgr/test.sh

```diff
@@ -262,7 +262,7 @@ testConfig(){
 	CFG=$OLDCFG
 
 	# $CFG/data cannot be empty
-	testFail find "$CFG/data" -maxdepth 0 -type d -empty
+	testFail [ -d "$CFG/data" ]
```

Comment:
Here, the directory can be empty where as before the directory cannot. Is this intended?

Reply:
Sorry, no clue. I will send Linus email to ask him about this apparent comment/code incompatibility when he gets back.

Reply:
Actually, I think I wrote the `find "$CFG/data" -maxdepth 0 -type d -empty` part. I did not want it to be empty because `scmgr` should write stuff into that folder. I'm not sure why Linus changed it.

Reply:
If there's no failure in the more strict version of the test, then we should change it back.

Reply:
OK, I've sent him email, he'll look at it when he's back.